packages/common/src/core/types/IHubProject.ts

```diff
@@ -17,14 +18,7 @@ export interface IHubProject
     IWithLayout,
     IWithMetrics,
     IWithPermissions {
-  status: PROJECT_STATUSES;
-}
-
-export enum PROJECT_STATUSES {
```

Comment:
Technically removing this is a breaking change. I'm not sure how strict we are about breaking changes, since it's likely this is only being used internally and we can replace our own instances in `opendata-ui`; however, an alternative to releasing a breaking change is to mark this as deprecated and direct people to start using the new `HubEntityStatus`, and then we can remove this during our next planned breaking change

Reply:
That's a good point. It's likely safe but keeping it and marking it deprecated is a good idea.

Reply:
Sorry if I was not clear in our chat about this @vivzhang - we should keep the PROJECT_STATUSES enum but mark as deprecated.